collections/St_Johns_College/SW_V2_St_Johns_College_MS_079.xml

```diff
@@ -0,0 +1,201 @@
+                           </dimensions> between columns, in 25 lines to the column. Frequent
+                           prickings, especially late in the volume; bounded and ruled in black ink.
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote script="textualisNorthern">Written in <term type="script">gothic
+                           textura rotunda</term> (without ‘hooks’, but the minims very narrowly
+                        spaced in relation to their height). Punctuation by <term
+                           type="punctuation>">point</term>, <term type="punctuation"
+                        >virgula</term>, <term type="punctuation">double point</term>, and <term
+                           type="punctuation">punctus interrogativus.</term></handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="rubrication">Headings in red for books and chapter
+                        numbers.</decoNote>
+                     <decoNote type="flourInit">At the head of each Gospel, a 5-line capital in red,
+                        blue, and violet on flourishing of the same with extended flourishing and
+                        bar to make a vinet.</decoNote>
+                     <decoNote type="flourInit">At chapter heads, 2-line blue lombards on red
+                        flourishing.</decoNote>
+                     <decoNote type="slashInit">Some ochre-slashed capitals to divide the
+                        text.</decoNote>
+                     <decoNote type="paratext">Red running titles indicate the biblical
+                        book.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <p>A modern replacement. Sewn on four thongs. At the front, two modern paper
+                        flyleaves; at the rear, fifteen modern paper leaves (iii–xvii).</p>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notBefore="1400" notAfter="1425">s. xv
+                        in.</origDate>
+                     <origPlace>
+                        <country key="place_7002445">England</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>'<persName key="person_4614" role="sgn">Thomas T<gap unit="char"
+                           quantity="1"/>y<gap unit="char" quantity="4"/></persName>', an erased
+                     ex-libris <locus from="66v" to="66v">(fol. 66<hi rend="superscript"
+                        >v</hi>).</locus></provenance>
+                  <provenance notBefore="1500" notAfter="1525">'<persName key="person_4615"
+                        role="fmo">Thomas Blownt</persName>' <locus from="32r" to="32r">(fol.
```

Comment:
it's not necessary to encode Blunt's name on every occurrence (unless his role changes) and may save time not to